src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -123,7 +124,7 @@ public ODataUtf8JsonWriter(Stream outputStream, bool isIeee754Compatible, Encodi
             this.bufferFlushThreshold = 0.9f * this.bufferSize;
             this.leaveStreamOpen = leaveStreamOpen;
             this.outputEncoding = encoding;
-            this.encoder = encoder ?? JavaScriptEncoder.Default;
+            this.encoder = encoder ?? JavaScriptEncoder.UnsafeRelaxedJsonEscaping;

```

Comment:
What are the limitations or risks of using this encoder? With SFI in mind😂.

Reply:
I've mentioned this in the PR description, let me quote it here:

> Justification for changing the default encoder to JavaScriptEncoder.UnsafeRelaxedJsonEscaping:

> None of the built-in encoder matches exactly any of the ODataStringEscapeOption used by JsonWriter. So there's no reason to prefer JavaScriptEncoder.Default based on consistency. It escapes more characters than either of the string escape options available for JsonWriter.
Performance: serializing strings that need to be escaped uses more CPU and allocates more memory than strings that don't need escaping. Therefore, an encoder that is less likely to escape strings is more efficient.
This encoder is used by default in AspNetCore, therefore this is more consistent with JSON-based APIs written in modern ASP.NET Core.
The security concerns that make JavaScriptEncoder.Default (e.g. escaping HTML special characters) do not apply if we don't dump the output directly in a HTML document. Since our use case is REST APIs, it's up to the client to escape the response if they intend to render their contents in a web page.